<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newProfessorDialog</class>
 <widget class="QDialog" name="newProfessorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblAvailability">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 14px;</string>
   </property>
   <property name="text">
    <string>Disponibilidad</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>110</y>
     <width>371</width>
     <height>227</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="grdAvailability">
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QHeaderView::section {
    background-color: rgb(11, 117, 250);
    height: 32px;
	border-bottom-style: dotted;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnAccept">
        <property name="styleSheet">
         <string notr="true">QPushButton {
      background-color: rgb(11, 117, 250);
	  border-style: outset;
	  border-width: 2px;
	  border-radius: 10px;
	  border-color: beige;
	  font: bold 14px;
	  color: white;
	  min-width: 10em;
	  padding: 6px;
 }


QPushButton:hover:!pressed
{
      background-color: rgb(3, 7, 128);
	  border-style: outset;
	  border-width: 2px;
	  border-radius: 10px;
	  border-color: beige;
	  font: bold 14px;
	  color: white;
	  min-width: 10em;
	  padding: 6px;
}
</string>
        </property>
        <property name="text">
         <string>Aceptar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExit">
        <property name="styleSheet">
         <string notr="true">QPushButton {
      background-color: rgb(250, 78, 78);
	  border-style: outset;
	  border-width: 2px;
	  border-radius: 10px;
	  border-color: beige;
	  font: bold 14px;
	  color: white;
	  min-width: 10em;
	  padding: 6px;
 }


QPushButton:hover:!pressed
{
      background-color: rgb(204, 0, 0);
	  border-style: outset;
	  border-width: 2px;
	  border-radius: 10px;
	  border-color: beige;
	  font: bold 14px;
	  color: white;
	  min-width: 10em;
	  padding: 6px;
}</string>
        </property>
        <property name="text">
         <string>Salir</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>324</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lblName">
      <property name="styleSheet">
       <string notr="true">font: bold 14px;</string>
      </property>
      <property name="text">
       <string>Nombre:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="txtName"/>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="cmbSubject">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>321</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Materia</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
